spec/row-chart-spec.js

```diff
@@ -352,11 +352,30 @@ describe('dc.rowChart', function () {
                     });
                 });
             });
+
+            if (xAxisTicks) {
+                describe('with elasticX', function () {
+                    beforeEach(function () {
+                        chart.elasticX(true)
+                            .xAxis().ticks(3);
+
+                        chart.render();
+                    });
+
+                    it('should generate x axis domain dynamically', function () {
+                        var nthText = function (n) { return d3.select(chart.selectAll('g.axis .tick text')[0][n]); };
+
+                        for (let i = 0; i < xAxisTicks.length; i++) {
```

Comment:
Crazy... all your code works well, but for some reason the `let` here caused Jasmine to skip the row chart tests! Completely silent failure!

dc.js is still ES5 and will be for a while yet. @sebgrohn, did you have to do anything special to get the tests to run?

Of course, I noticed 58 commits later that my test count was way down. Easy enough to fix, but whoah that's a little scary if one ES6 keyword causes a file to get dropped entirely.